src/components/cards/standard/StandardCard.tsx

```diff
@@ -8,7 +8,8 @@ import { StandardCardDisplay, StandardCardCssClasses } from './StandardCardDispl
  *
  * @public
  */
-export interface StandardCardProps extends CardProps {
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export interface StandardCardProps extends CardProps<any> {
```

Comment:
There's are type conflict issues when using StandardCard is a default for a `CardComponent` prop. For instance, in StandardSection, the generic type T would apply to `CardComponent` prop and `results` in props to be T. When CardComponent is not given, if StandardCard is used as the default with a type of DefaultRawDataType, we would have `results` with type T and a `CardComponent` with type DefaultRawDataType, causing a conflict since Type 'T' is not assignable to type 'DefaultRawDataType'. So, the generic type for StandardCardProps needs to be any for the card to be use as default.

<img width="806" alt="Screen Shot 2022-07-25 at 9 27 52 AM" src="https://user-images.githubusercontent.com/36055303/180788678-0c491127-70b3-47e4-a542-85316478b184.png">